.drone.yml

```diff
@@ -56,7 +56,7 @@ pipeline:
       event: [ push, tag, pull_request ]
 
   build-without-gcc:
-    image: golang:1.9
+    image: golang:1.10
```

Comment:
Shouldn't this also be `1.12`?

Reply:
this step is kept as the lowest version of golang that we support. Currently we support all the way down to 1.9 but this changes it to be 1.10 be the oldest we support.